br/pkg/task/backup.go

```diff
@@ -365,6 +358,10 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	if err != nil {
 		return errors.Trace(err)
 	}
+	// if use noop as external storage, turn off the checkpoint mode
+	if u.GetNoop() != nil {
+		cfg.UseCheckpoint = false
```

Comment:
Please add a message notifying that the checkpoint is ignored for xxx.

Reply:
done